Components/Metrics/Impact/elxImpactMetric.hxx

```diff
@@ -0,0 +1,456 @@
+/*=========================================================================
+ *
+ *  Copyright UMC Utrecht and contributors
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0.txt
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ *=========================================================================*/
+
+#ifndef elxImpactMetric_hxx
+#define elxImpactMetric_hxx
+
+#include "elxImpactMetric.h"
+#include "itkTimeProbe.h"
+#include <vector>
+#include <sstream>
+#include <filesystem>
```

Comment:
@vboussot For your information, I asked at the ITK forum whether it is OK nowadays to use `std::filesystem`. And I received quite some interesting feedback: https://discourse.itk.org/t/shall-we-start-using-c-17-at-the-master-branch/5828/23  I concluded that for elastix, it should be OK 👍